build.sbt

```diff
@@ -197,7 +192,7 @@ lazy val userImpl = (project in file("user-impl"))
 
 lazy val webGateway = (project in file("web-gateway"))
   .settings(commonSettings: _*)
-  .enablePlugins(PlayJava && LagomPlay)
+  .enablePlugins(PlayJava && LagomPlay, SbtReactiveAppPlugin)
```

Comment:
is `PlayJava && LagomPlay, SbtReactiveAppPlugin` equivalent to `PlayJava, LagomPlay, SbtReactiveAppPlugin`?? If yes, can we use a single syntax?